doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -92,15 +92,16 @@ any kind of capacity planning.
 
 <!-- RELATION_SPEC mz_catalog.mz_cluster_replicas -->
-Field               | Type      | Meaning
---------------------|-----------|--------
-`id`                | [`text`]  | Materialize's unique ID for the cluster replica.
-`name`              | [`text`]  | The name of the cluster replica.
-`cluster_id`        | [`text`]  | The ID of the cluster to which the replica belongs. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters).
-`size`              | [`text`]  | The cluster replica's size, selected during creation.
-`availability_zone` | [`text`]  | The availability zone in which the cluster is running.
-`owner_id`          | [`text`]  | The role ID of the owner of the cluster replica. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).
-`disk`              | [`boolean`] | If the replica has a local disk.
+Field                     | Type        | Meaning                                                                                                                                                         |
+--------------------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                      | [`text`]    | Materialize's unique ID for the cluster replica.                                                                                                              |
+| `name`                    | [`text`]    | The name of the cluster replica.                                                                                                                              |
+| `cluster_id`              | [`text`]    | The ID of the cluster to which the replica belongs. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters).                           |
+| `size`                    | [`text`]    | The cluster replica's size, selected during creation.                                                                                                         |
+| `availability_zone`       | [`text`]    | The availability zone in which the cluster is running.                                                                                                        |
+| `owner_id`                | [`text`]    | The role ID of the owner of the cluster replica. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).                                    |
+| `disk`                    | [`boolean`] | If the replica has a local disk.                                                                                                                              |
+| `pending_reconfiguration` | [`boolean`] | If the replica was created from a managed cluster reconfiguration that has not yet completed. The configuration of this replica may differ from the cluster.  |
```

Comment:
Minor naming nit, but requesting changes so it doesn't get lost: I think we should avoid using the term "reconfiguration" in user-facing docs since it's not a term users are likely to be familiar with or understand quickly. I'd advocate for `pending_alter` here instead.

Reply:
If we plan to use graceful reconfig for auto-scaling, which we do, will `pending_alter` be misleading.

Reply:
I think it'll be the opposite! The way I would want to document autoscaling, at least, is that it "automatically alters the size of your cluster based on load." And we'll also need to figure out how a manual `ALTER` would interact with autoscaling. So I think it all ties together nicely with a name like `pending_alter`.

Reply:
That said, this is going to insta-stabilize, which is always a bit scary. The safest thing to do is probably to create a new relation called `mz_unstable_cluster_replicas` and put the `pending_alter` column in there. That will give us latitude to revisit during autoscaling if it does turn out to be confusing.